robottelo/cli/factory.py

```diff
@@ -1298,28 +1298,6 @@ def make_host(options=None):
         provision                                   true/false
         virtual                                     true/false
     """
-    # Check for required options
```

Comment:
these are not required options for all situations. For example, we can specify all necessary parameters in host group and then inherit them into host